go/cmd/dolt/cli/documentation_helper.go

```diff
@@ -199,31 +190,21 @@ var CliFormat = docFormat{"<", ">", "<b>", "</b>"}
 // Special format for the synopsis which is rendered inside raw HTML in markdown
 var SynopsisMarkdownFormat = docFormat{"&lt;", "&gt;", "`", "`"}
 
-func transformSynopsisToHtml(commandStr string, synopsis []string) string {
+func transformSynopsisToMarkdown(commandStr string, synopsis []string) string {
```

Comment:
i don't understand the context of why you're switching from HTML to markdown here, I'm assuming you know what you're doing.  afaik most of this PR just looks like `dolt --help` printing, were we gatsby formatting those before?

Reply:
tests would have made it cleared where this is used

Reply:
It was always markdown, just markdown with embedded HTML